### DISCUSSION TITLE
Better handling of underlined text

### DISCUSSION THREAD

**Discussion by problem** [outline/outline#6291] [Ideas]
I've run into this problem many, many times now. I'd call it a minor nuisance / quality of life issue. I copy text from somewhere else into Outline. If it's a link, it usually includes an underline already. Outline thinks this is a standard text decoration, and picks it up, but it cannot be removed. If it's just underlined text, the same happens. It comes over to outline, but then you are stuck with the underline.

I'd love to see either: underlines stripped out completely, or an underline option in formatting so that it can be deselected.

<img width="499" alt="Screenshot 2023-12-14 at 12 27 19 PM" src="https://github.com/outline/outline/assets/3143/97a84879-7e3f-43b4-8755-e6e441cc444e">

Of course, the workaround is just to paste the text into a plain text editor first to strip the formatting, and then to Outline.

**Comment by tommoor** [comment]
Created an [issue](https://github.com/outline/outline/issues/6292), agreed - it can be stripped _and_ we should show an underline toggle in this circumstance.

FWIW you can toggle underline with the keyboard shortcut `CMD+U`
